hera_cal/vis_clean.py

```diff
@@ -786,6 +787,76 @@ def fft_data(data, delta_bin, wgts=None, axis=-1, window='none', alpha=0.2, edge
     return data, fourier_axes
 
 
+def trim_model(clean_model, clean_resid, dnu, keys=None, noise_thresh=5.0, delay_cut=3000,
```

Comment:
Why 5.0?

Reply:
just b/c its safer than 1.0, no particular reason, we could do 2.0 as well.